File: app2/src/main/java/ai/meccamedinatv/mekkalive/online/mvp/view/LoaderView.java
package ai.meccamedinatv.mekkalive.online.mvp.view;

import moxy.MvpView;
import moxy.viewstate.strategy.AddToEndSingleTagStrategy;
import moxy.viewstate.strategy.StateStrategyType;

@StateStrategyType(AddToEndSingleTagStrategy.class)
public interface LoaderView extends MvpView
{

    @StateStrategyType(AddToEndSingleTagStrategy.class)
    void progressDialogHide();

    @StateStrategyType(AddToEndSingleTagStrategy.class)
    void progressDialogShow();
}

File: ytlib/src/main/java/com/walhalla/ytlib/mvp/LoaderView.kt
package com.walhalla.ytlib.mvp

import moxy.MvpView
import moxy.viewstate.strategy.AddToEndSingleTagStrategy
import moxy.viewstate.strategy.StateStrategyType

@StateStrategyType(AddToEndSingleTagStrategy::class)
interface LoaderView : MvpView {
    @StateStrategyType(AddToEndSingleTagStrategy::class)
    fun progressDialogHide()

    @StateStrategyType(AddToEndSingleTagStrategy::class)
    fun progressDialogShow()
}
